package.json

```diff
@@ -17,10 +17,12 @@
     "cy:run": "cypress run"
   },
   "dependencies": {
+    "@next/font": "^13.0.5",
```

Comment:
Had problems running the code with this version of "`@next/font`". Got a warning about the different version of `@next/font` and `next`:

<img width="883" alt="Screenshot 2022-12-01 at 19 05 01" src="https://user-images.githubusercontent.com/2343039/205138249-6494fbd7-9451-4ad4-b237-13a398037a87.png">

And the app failed to load in the browser:

![Screenshot 2022-12-01 at 19 03 57](https://user-images.githubusercontent.com/2343039/205138135-ce361bec-0b05-4175-b1e5-8145d5c37743.png)

After upgrading to `"next": "13.0.5"` it worked just fine.